libeos-payg/util.c

```diff
@@ -34,20 +34,18 @@ static gboolean payg_legacy_mode = FALSE;
 __attribute__ ((__noreturn__)) gboolean
 payg_sync_and_poweroff (gpointer user_data)
```

Comment:
This function is now misleadingly named.

Reply:
It still calls poweroff, but yeah, I agree it could be better named.